src/Sarif/CodeGenHints.json

```diff
@@ -506,6 +496,29 @@
       }
     }
   ],
+  "Result.Level": [
+    {
+      "kind": "EnumHint",
+      "arguments": {
+        "typeName": "FailureLevel",
```

Comment:
>FailureLevel [](start = 21, length = 12)

Wow. Is this the first time we've had two properties need the same generated enum type, and had the problem that now we have to keep them in sync? Oh well. Ideally the code hint processor would provide a way for an `EnumHint` to indirectly specify a common enum type, but today is not that day.

Reply:
Oh wow, `RuleConfiguration.DefaultLevel` below uses the same type -- it now appears three times in this file. Good luck to us.

---
In reply to: [252889661](https://github.com/Microsoft/sarif-sdk/pull/1249#discussion_r252889661) [](ancestors = 252889661)